content/tagging/_index.md

```diff
@@ -17,19 +17,19 @@ further_reading:
 
 ## Introduction
 
-Tags are a way of adding dimensions to metrics, so they can be sliced, diced, aggregated, and compared on the front end. [Using tags][1] enables you to observe aggregate performance across a number of hosts and (optionally) narrow the set further based on specific elements. In a nutshell, tagging is a method to scope aggregated data.
+Tags are a way of adding dimensions to metrics, so they can be sliced, diced, aggregated, and compared on the front end. [Using tags][1] enables you to observe aggregate performance across a number of hosts and (optionally) narrow the set further based on specific elements. In summary, tagging is a method to scope aggregated data.
```

Comment:
We are forgetting a big concept around tagging here. 

**Tagging is the key to bind different data types in Datadog**, allowing for correlation and call to action between metrics, traces, and logs in the DD app.

We are specifying in the next section that some tags are reserved tags key (`host`, `service`, and `source`) but we never explain **why** (btw, idk why the `device` tag is a reserved tag.. we should investigate on this). I think we should emphasize those concepts here:

- Explaining that the `service` tag allows for correlation between APM and logs.
- Explaining  that the `host` tag allows for correlation between metrics, traces, processes, and logs
- Explaining  that the `source` tag allows for automated pipeline creation for the log-management solution 

We should also explain here why using the DD Agent is so important => because it assures consistency of tag applied to metrics/traces/logs/processes. Especially for the `host` tag since the agent is able to determine the hostname with a good accuracy: https://docs.datadoghq.com/agent/faq/how-datadog-agent-determines-the-hostname/

Reply:
I believe `device` is still reserved (used in the disk check). I was able to find a recent ticket that supports this - https://datadog.zendesk.com/agent/tickets/163625

Checking on the list with cake and who to contact for verification.